pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -219,6 +223,18 @@ type CommonPrometheusFields struct {
 	// Number of seconds to wait until a scrape request times out.
 	ScrapeTimeout Duration `json:"scrapeTimeout,omitempty"`
 
+	// The protocols to negotiate during a scrape. It tells clients what
+	// protocol are accepted by Prometheus and with what weight (most wanted is first).
+	// Supported values (case sensitive): PrometheusProto, OpenMetricsText0.0.1,
+	// OpenMetricsText1.0.0, PrometheusText0.0.4.
+	//
+	// If unset, Prometheus uses its default value: ["OpenMetricsText1.0.0","OpenMetricsText0.0.1","PrometheusText0.0.4"]
+	//
+	// It requires Prometheus >= v2.48.0.
+	//
+	// +optional
```

Comment:
(nit): `+listType=atomic`

Reply:
Thanks for your review. Done.

Reply:
+listType was new to me, could you explain why choose atomic here?

[Reading the docs](https://book.kubebuilder.io/reference/markers/crd-processing) I'd say it is a set 🤔, but again, this is new to me
<img width="744" alt="image" src="https://github.com/prometheus-operator/prometheus-operator/assets/24193764/b4aa87fc-f570-462a-a142-81f67f982a91">

Reply:
https://github.com/kubernetes/kube-openapi/blob/master/pkg/idl/doc.go  `atomic` as default value for enum